src/InputContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { addTask, updateTaskTitle } from './store/actions';
+import Input from './Input';
```

Comment:
```suggestion
import { useDispatch, useSelector } from 'react-redux';

import { addTask, updateTaskTitle } from './store/actions';

import Input from './Input';
```

관련이 있는 것들끼리 적절하게 묶어 주는게 좋을 것 같습니다

Reply:
넵!